Base.rte/Scenes/Objects/Bunkers/BunkerSystems/Ladders/Ladders.ini

```diff
@@ -131,4 +150,38 @@ AddTerrainObject = TerrainObject
 			X = 12
 			Y = 12
 		HFlipped = 1
-			
\ No newline at end of file
+
+
+AddTerrainObject = TerrainObject
+	PresetName = Background Ladder
+	AddToGroup = Bunker Systems
+	GoldValue = 10
+	FGColorFile = ContentFile
+		Path = Base.rte/Scenes/Objects/Bunkers/BunkerSystems/Ladders/LadderCFG.bmp
```

Comment:
No need for these dupe empty FG and Mat sprites.
Just use these instead and remove the dupes.
```
Base.rte/Scenes/Objects/Bunkers/BunkerBackgrounds/Empty1XFG.bmp
Base.rte/Scenes/Objects/Bunkers/BunkerBackgrounds/Empty1XMat.bmp
```